CMakeLists.txt

```diff
@@ -1646,18 +1801,34 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/wolfssl/
         DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/wolfssl
         FILES_MATCHING PATTERN "*.h"
         REGEX ${EXCLUDED_HEADERS_REGEX} EXCLUDE)
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/cyassl/
+        DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/cyassl
+        FILES_MATCHING PATTERN "*.h"
+        REGEX ${EXCLUDED_HEADERS_REGEX} EXCLUDE)
+
+
 # Install the examples
 install(FILES ${INSTALLED_EXAMPLES}
         DESTINATION ${CMAKE_INSTALL_DOCDIR}/example)
 # Install README.txt and taoCert.txt
 install(FILES
         ${CMAKE_CURRENT_SOURCE_DIR}/doc/README.txt
         ${CMAKE_CURRENT_SOURCE_DIR}/certs/taoCert.txt
-        DESTINATION ${CMAKE_INSTALL_DOCDIR}/wolfssl)
+        DESTINATION ${CMAKE_INSTALL_DOCDIR})
 # Install the export set
 install(EXPORT wolfssl-targets
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/wolfssl
         FILE wolfssl-config.cmake)
 
 # TODO: Distro build + rules for what to include in the distro.
```

Comment:
Can we remove this comment with your changes here?

Reply:
I'm not sure if that fully addresses the TODO. I think there is still work around the `--enable-distro` option.